### DISCUSSION TITLE
能否将loading加载动画的头像变为圆形

### DISCUSSION THREAD

**Discussion by Aaron-zsy** [hexo-theme-Acrylic/Hexo-Theme-Acrylic#60] [Q&A]
![{D311292C-2875-4614-BE14-10EE416BA365}](https://github.com/hexo-theme-Acrylic/Hexo-Theme-Acrylic/assets/122505440/143386ee-01f1-4046-9a24-ac7b89367a81)

**Comment by ghost** [comment]
如果是pro的话应该就是圆形吧，加649565726咨询吧，方便一点
